data/human/marauders.txt

```diff
@@ -533,15 +541,16 @@ ship "Marauder Leviathan" "Marauder Leviathan (Engines)"
 	add attributes
 		"engine capacity" 40
 	outfits
-		"Electron Beam" 4
-		"Quad Blaster Turret" 4
+		"Heavy Rocket Launcher" 4
+		"Heavy Rocket" 80
+		"Gatling Turret" 4
+		"Gatling Gun Ammo" 24000
 		
 		"Fusion Reactor"
 		"LP144a Battery Pack"
-		"D94-YV Shield Generator"
-		"D67-TM Shield Generator"
+		"D94-YV Shield Generator" 2
+		"Supercapacitor" 5
```

Comment:
```suggestion
		"LP036a Battery Pack"
```

Reply:
Or, alternatively, since it already has an LP144a battery pack you could replace them with security stations.

Reply:
```suggestion
		"Security Station" 5
```
yeah I like that better